external/llvm-project/mlir/lib/Conversion/GPUCommon/GPUOpsLowering.cpp

```diff
@@ -173,10 +173,9 @@ static LLVM::LLVMFuncOp getOrDefineFunction(T &moduleOp, const Location loc,
 }
 
 LogicalResult GPUPrintfOpToHIPLowering::matchAndRewrite(
-    gpu::PrintfOp gpuPrintfOp, ArrayRef<Value> operands,
```

Comment:
Apparently the OpAdaptor form is preferred and the ArrayRef form is becoming deprecated.  I changed these because otherwise matchAndRewrite() wasn't being found when compiling.

Reply:
Yep, those changes are incorporated upstream